File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int n = nums.length;
        int zero=0, one=0, two=0;
        for(int i=0; i<n; i++){
            if(nums[i]==0){
                zero++;
                continue;
            }
            if(nums[i]==1){
                one++;
                continue;
            }
            if(nums[i]==2){
                two++;
                continue;
            }
        }
        for(int i=0; i<n; i++){
            if(i<zero){
                nums[i]=0;
            }
            else if(i<(zero+one)){
                nums[i]=1;
            }
            else{
                nums[i]=2;
            }
        }   
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        zero, one, two = 0, 0, 0
        for num in nums:
            if(num==0):
                zero+=1
            elif(num==1):
                one+=1
            else:
                two+=1
        
        i=0
        while i<zero:
            nums[i]=0
            i+=1
        while i<(one+zero):  
            nums[i]=1
            i+=1
        while i<(one+zero+two):
            nums[i]=2
            i+=1